src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java

```diff
@@ -77,9 +77,9 @@ private static synchronized HttpClient getHttpClient() {
         if (httpClient == null) {
             final AsyncHttpClient asyncHttpClient =
                 new DefaultAsyncHttpClient(
-                    new DefaultAsyncHttpClientConfig.Builder().setAcceptAnyCertificate(true)
-                                                              .setHandshakeTimeout((int) DEFAULT_TIMEOUT)
-                                                              .build());
+                    new DefaultAsyncHttpClientConfig.Builder()
+                                                    .setHandshakeTimeout((int) DEFAULT_TIMEOUT)
```

Comment:
Deprecated method in the new version of the AsyncHttpClient used by the new JVM SDK version. 

This property is not needed anyways. I remember me and Andrii added it a long time ago when we thought we sometimes got errors from requests to the CTP. But that was a problem of not having a retry client. Now, I guess its not needed. Especially since its now deprecated from this new version used underneath by the JVM SDK.